app/controllers/games_controller.rb

```diff
@@ -32,19 +34,16 @@ def edit
   # POST /games
   # POST /games.json
   def create
-    @game = Game.new(game_params)
-    @game.current_turn = 0
-    @game.black_player_id = current_player.id if @game.white_player_id == 0
-    
-    respond_to do |format|
+    ActiveRecord::Base.transaction do
```

Comment:
Thanks @lasley and @montague for the detailed convo, I have refactored to put this stuff in the model. everything seems to be working as planned so that is a plus.